androidApp/src/main/java/press/editor/EditorActivity.kt

```diff
@@ -49,7 +49,9 @@ class EditorActivity : ThemeAwareActivity() {
     Observable.timer(delayFocus, MILLISECONDS, mainThread())
         .takeUntil(editorView.detaches())
         .subscribe {
-          editorView.editorEditText.showKeyboard()
+          if (intent.hasExtra(Intent.EXTRA_TEXT).not()) {
+            editorView.editorEditText.showKeyboard()
+          }
```

Comment:
Let's move this behavior to the presenter so that other platforms (iOS in the future) can also implement it as part of the contract.